pages/style-guide/js.md

```diff
@@ -249,35 +249,42 @@ elements
 When an entire file is wrapped in a closure, the body of the closure is not indented.
 
 ```js
-(function( $ ) {
+( function( $ ) {
 
-// this doesn't get indented
+// This doesn't get indented
 
-})( jQuery );
+} )( jQuery );
 ```
 
+```js
+module.exports = function( grunt ) {
+
+// This doesn't get indented
+
+};
+
```

Comment:
Also missing closing backticks, messing up the highlighting of the remaining file...